src/player.cpp

```diff
@@ -150,6 +150,15 @@ void Player::Init(int argc, char *argv[]) {
 	// Must be called before the first call to Output
 	Graphics::Init();
 
+	// FIXME: actual command line parsing is too late for setting this,
+	// it should be refactored after release to not break things now
+	for (int i = 1; i < argc; ++i) {
+		if (strcmp(argv[i], "--no-log-color") == 0) {
+			Output::SetTermColor(false);
+			break;
```

Comment:
this is really a general problem that should be addressed in a further PR.

This is also a problem for --help and --version: Before printing there stuff the console already contains "EasyRPG" and "CLI".